### PR TITLE
Hi memory usage on realtime replication [JIRA: RCS-192]

### PR DESCRIPTION
**Pull request by kuenishi**
This issue does not fix anything, but adds a metric to see zd://10584 reproduce before things collapse. 

Also see No.2 item "Sink Helper Stuck" in https://github.com/basho/platform_tasks/pull/61 for detailed description of this issue.
I want this be backported to 2.0 branch, as well as merged up to develop branch.

### CODE REVIEW

**Comment by bsparrow435** [comment]
tested manually and produces expected output.

**Comment by bsparrow435** [comment]
:+1: 48539de7eabaf3b5dee581b7b162a46dde8ffdd4

**Comment by kuenishi** [comment]
@borshop merge

File: src/riak_repl_util.erl [hunk 1]
```diff
@@ -825,6 +826,13 @@ safe_pid_to_list(Pid) when is_pid(Pid) ->
 safe_pid_to_list(NotAPid) ->
     NotAPid.
 
+safe_get_msg_q_len(Pid) when is_pid(Pid) ->
+    ProcInfo = erlang:process_info(Pid),
```
Comment (jonmeredith) [inline src/riak_repl_util.erl:830]: Sorry for being late on this one... why do the extra work of requesting information you don't need?

```
{message_queue_len, Len} = erlang:process_info(Pid, message_queue_len),
Len
```

Comment (kuenishi) [inline src/riak_repl_util.erl:830]: Great, will follow with another pull request.

**Comment by kuenishi** [comment]
Opened #676 to follow @jonmeredith 's comment.

**Comment by Basho-JIRA** [comment]
Merged to 2.0 and 2.1. Will be in each of next release. 

**_[posted via JIRA by Kota Uenishi]**_

**Pull request merged**